packages/components/empty-state/stories/EmptyState.stories.tsx

```diff
@@ -12,14 +13,14 @@ export default {
 
-export const Basic = (args: EmptyStateProps) => <EmptyState {...args} />;
+export const basic: Story<any> = ({ ...args }) => <EmptyState {...args} />;
```

Comment:
I think these Pascal case names were right, it follows the Storybook convention.